File: crates/auto-lang/test/a2r/008_method/method.expected.rs
struct Point {
    x: i32,
    y: i32,
}

impl Point {
    fn modulus(&self) -> i32 {
        self.x * self.x + self.y * self.y
    }
}

fn main() {
    let p: Point = Point { x: 3, y: 4 };
    let m: i32 = p.modulus();
    println!("Modulus: {}", m);
}

File: crates/auto-lang/test/a2c/11_methods/001_method/method.expected.c
#include "method.h"

int Point_Modulus(struct Point *self) {
    return self->x * self->x + self->y * self->y;
}

int main(void) {
    struct Point p = {.x = 3, .y = 4};
    int m = Point_Modulus(&p);
    printf("%s %d\n", "Modulus:", m);
    return 0;
}
